src/background/background-init.ts

```diff
@@ -31,7 +35,7 @@ import { TelemetryLogger } from './telemetry/telemetry-logger';
 import { TelemetryStateListener } from './telemetry/telemetry-state-listener';
 import { UserStoredDataCleaner } from './user-stored-data-cleaner';
 
-declare var window: Window;
+declare const window: InsightsFeatureFlags & Window;
```

Comment:
Tiny optional comment: I think this would be a little clearer as `Windows & InsightsFeatureFlags` (it's a window with the feature flag property added to it...)